config/sys.config

```diff
@@ -33,10 +33,10 @@
         {block_sync_batch_limit, 100},
         {honor_quick_sync, true},
         {quick_sync_mode, blessed_snapshot},
-        {blessed_snapshot_block_height, 1108081},
+        {blessed_snapshot_block_height, 1130401},
         {blessed_snapshot_block_hash,
```

Comment:
there's a newer snapshot on s3 I believe? latest-snap.json shows that's the last one there

Reply:
that's the latest available from latest-snap.json but i can update to one of the later ones if you can get me the hash/height

Reply:
@abhay got this for me